### DISCUSSION TITLE
Prevent "edgedb migration create" from re organising the fields in alphabetical order ?

### DISCUSSION THREAD

**Discussion by rafoxme** [geldata/gel#3792] [Q&A]
hello is there a solution to prevent "edgedb migration create" from re organising the fields in alphabetical order ?
![image](https://user-images.githubusercontent.com/33931532/164981579-805ccd71-6738-4383-a54f-df4ee6bad97d.png)
![image](https://user-images.githubusercontent.com/33931532/164981585-6102dbe3-7af5-4942-8bdd-28ea2fa6528f.png)

**Comment by elprans** [comment]
The order of fields in a type is generally not defined and in many cases impossible to keep as-is due to possible functional dependencies.  Can you elaborate why you need this please?

**Comment by rafoxme** [comment]
Hi @elprans i just want to keep fields that have common utility grouped.
for example :
 **ceated_at** and the **updated_at**

**Reply by elprans** [reply]
Hm, I'm still not sure what you mean.  You can certainly keep the fields in the preferred order in your `.esdl` files.
